api/src/pcapi/alembic/versions/20230724T093020_c1c137b863fe_add_preferences_jsonb_to_educational_redactor.py

```diff
@@ -0,0 +1,25 @@
+"""
+add preferences (jsonb) to educational redactor
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "c1c137b863fe"
+down_revision = "97fcc93be70a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column(
+        "educational_redactor",
+        sa.Column("preferences", postgresql.JSONB(astext_type=sa.Text()), server_default="{}", nullable=True),
```

Comment:
Ne pourrait-on pas rendre le champ NOT NULLable, puisqu'il a une valeur par défaut ?

Reply:
Au temps pour moi, j'ai laissé passer ça. Je corrige.